keras_nlp/models/f_net/f_net_masked_lm.py

```diff
@@ -135,6 +136,12 @@ def __init__(
         # All references to `self` below this line
         self.backbone = backbone
         self.preprocessor = preprocessor
+        self.compile(
+            loss=keras.losses.SparseCategoricalCrossentropy(from_logits=True),
+            optimizer=keras.optimizers.Adam(2e-5),
+            metrics=keras.metrics.SparseCategoricalAccuracy(),
```

Comment:
Actually, I think we want weighted_metrics here as we do pass sample weights for this task.

Reply:
Sure

Reply:
Still working on the converging part